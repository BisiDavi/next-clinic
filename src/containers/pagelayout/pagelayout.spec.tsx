import React from 'react'
import { render } from '@testing-library/react'
import Pagelayout from './pagelayout'

describe('Pagelayout', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<Pagelayout />)
    expect(asFragment()).toMatchSnapshot()
  })

  it('check classname', () => {
    const { getByTestId } = render(
      <Pagelayout>
        <p>Test me</p>
      </Pagelayout>,
    )

    const TEST_ID = getByTestId('pagelayout').className

    expect(TEST_ID).toBe('container')
  })
})
