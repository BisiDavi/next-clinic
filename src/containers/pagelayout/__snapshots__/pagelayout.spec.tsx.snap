// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagelayout renders correctly 1`] = `
<DocumentFragment>
  <div
    class="container"
    data-testid="pagelayout"
  >
    <div>
      <h1
        class="header"
        data-testid="header"
      >
        Header Page
      </h1>
    </div>
    <div>
      <h1>
        Footer page
      </h1>
    </div>
  </div>
</DocumentFragment>
`;
